frontend/src/components/Logs/CategoryHeading/CategoryHeading.styles.scss

```diff
@@ -0,0 +1,3 @@
+.container--text-size {
```

Comment:
if this global class use this one in global styles rather than declaring here

Reply:
This is not a global class, will update the name.